### DISCUSSION TITLE
raspicat_documentation/chat_corner/error_resolution/raspicat_error_resolution/

### DISCUSSION THREAD

**Discussion by giscus** [CIT-Autonomous-Robot-Lab/raspicat_documentation#14] [Announcements]
# raspicat_documentation/chat_corner/error_resolution/raspicat_error_resolution/

https://uhobeike.github.io/raspicat_documentation/chat_corner/error_resolution/raspicat_error_resolution/

**Comment by TonoLeo** [comment]
sudo apt upgradeでCould not get lock /var/lib/dpkg/lock-frontend. It is held by process　数字　というエラー

**Reply by TonoLeo** [reply]
sudo kill -9 数字（エラーに出てきたのを入れる）を実行しもう一度、sudo apt upgradeをする
https://mebee.info/2021/05/26/post-34983/

**Comment by TonoLeo** [comment]
ssh-copy-id ubuntu@10.42.0.XX
上のpassをコピーするコマンドを入力して　/usr/bin/ssh-copy-id: ERROR: No identities found
このエラーが出たときはgit のsshキーの登録ができていないためのエラーなので公開鍵の設定をしてからコマンドを入力してください
